nf_core/lint/files_exist.py

```diff
@@ -235,7 +240,23 @@ def pf(file_path):
             failed.append(f"File must be removed: {self._wrap_quotes(file)}")
         else:
             passed.append(f"File not found check: {self._wrap_quotes(file)}")
-
+    # Files that cause an error if they exists together with a certain entry in nextflow.config
+    for file in files_fail_ifinconfig:
+        if file[0] in ignore_files:
+            continue
+        nextflow_config = pf("nextflow.config")
+        in_config = False
+        with open(nextflow_config) as f:
+            if file[1] in f.read():
+                in_config = True
+        if os.path.isfile(pf(file[0])) and in_config:
+            failed.append(f"File must be removed: {self._wrap_quotes(file[0])}")
+        elif os.path.isfile(pf(file[0])) and not in_config:
+            passed.append(f"File not found check: {self._wrap_quotes(file[0])}")
```

Comment:
This message confused me a little bit, would it be better to remove the `not`?
```suggestion
            passed.append(f"File found check: {self._wrap_quotes(file[0])}")
```